web/templates/index.html

```diff
@@ -19,6 +19,12 @@
   <link rel="stylesheet" href="/static/css/font-awesome.min.css"/>
 </head>
 <body>
+  <script src="https://cdn.ravenjs.com/2.3.0/raven.min.js"></script>
+  <script>
+    // Setup sentry reporting
+    Raven.config('https://fd20ee7ca8f7421fb4cddbd489117f22@app.getsentry.com/73224').install()
```

Comment:
Should this token be stored in `secrets.py` and rendered into the template in production (see also: #122)? It's not a "secret", per se, but it is a value that would fit well with other configuration management.